### ISSUE TITLE
`deno bundle` should keep the shebang in the bundled file

### ISSUE THREAD

**Issue by juanfra684** [denoland/deno#16936]
I would like to use `deno bundle` to generate CLI scripts without dependencies. All works fine but `deno bundle` generates files with the shebang removed. Here is an example:
```plain
$ cat test.js 
#!/usr/bin/env -S deno run --allow-read

for (const item of Deno.readDirSync(".")) {
    console.log(item.name);
}
$ cat test.bundle.js 
// deno-fmt-ignore-file
// deno-lint-ignore-file
// This code was bundled using `deno bundle` and it's not recommended to edit it manually

for (const item of Deno.readDirSync(".")){
    console.log(item.name);
}
```

**Comment by zhenfeng-zhu** [comment]
Let me try to implement this feature.

**Comment by GJZwiers** [comment]
Fixed in 1.29.2
